installer/templates/new/web/views/error_view.ex

```diff
@@ -13,5 +13,17 @@ defmodule <%= application_module %>.ErrorView do
   # template is found, let's render it as 500
   def template_not_found(_template, assigns) do
     render "500.html", assigns
+  end<% else %>def render("404.json", _assigns) do
+    %{"errors" => %{"detail" => "Page not found"}}
   end
+
+  def render("500.json", _assigns) do
+    %{"errors" => %{"detail" => "Server internal error"}}
```

Comment:
> Don't we recommend string keys for payloads in the channel layer? I would return whatever the channel layer returns.

Not in our doc examples. My general rule is string keys coming in from the outside, and atoms going out